docker/bin/activate

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
If causes an issue we can still commit and get back to it later. Let me know :) Thanks so much for improving this! Appreciated! :1st_place_medal: